mkdocs.yml

```
@@ -20,7 +20,8 @@ nav:
   - Overview: index.md
   - Work with documentation: mkdocs.md
   - Jobs and tasks: states.md
-  - Simulator binaries: simulator_binaries.md
+  - Building backend for developers: for_developers.md
+  - Building backend for deployment using Docker: using_docker.md
```

```suggestion
  - Building and run using Docker: using_docker.md
```

whole documentation here is for developers :)